{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -306,8 +306,12 @@
 ACCOUNT_EMAIL_VERIFICATION = "mandatory"
 # https://django-allauth.readthedocs.io/en/latest/configuration.html
 ACCOUNT_ADAPTER = "{{cookiecutter.project_slug}}.users.adapters.AccountAdapter"
+# https://django-allauth.readthedocs.io/en/latest/forms.html
+ACCOUNT_FORMS = {"signup": "{{cookiecutter.project_slug}}.users.forms.UserSignupForm"}
```

Comment:
While looking through this, I ended up looking at the allauth code and found out about [the setting](https://django-allauth.readthedocs.io/en/latest/configuration.html?highlight=ACCOUNT_SIGNUP_FORM_CLASS) `ACCOUNT_SIGNUP_FORM_CLASS`. If this setting is configured, the class is used as base class for both `SignupForm` (social and not social). 

It seems like we could configure this single setting instead of these 2. What do you think?

Reply:
Isn't it possible that some of the information that will fill up the profile of a new user will come from social information? 

I am not experienced in social logins/signups but as a user, some information might be requested by the app, like phone number, birth date, etc. that would make the journey for a user signing up from social quite different from one that signup using email/password.

If it is indeed the case, having separate classes for social and non-social would make more sense compared to a single unified form. That might also have been the reason why allauth has given somewhat more attention to separate forms than to a combined single signup form.

What do you think?

Reply:
That's a good point, probably best to stick to the current approach 👍🏻